cpp/installer.cc

```diff
@@ -567,10 +572,8 @@ void InstallVuFindCronjobs(const VuFindSystemType vufind_system_type) {
         names_to_values_map.insertScalar("relbib_host", GetStringFromTerminal("RelBib Hostname"));
     }
 
-    if (vufind_system_type == KRIMDOK)
-        InstallCronjobs("krimdok.cronjobs", start_vufind_autogenerated, end_vufind_autogenerated);
-    else
-        InstallCronjobs("ixtheo_cronjobs", start_vufind_autogenerated, end_vufind_autogenerated, names_to_values_map);
+    InstallCronjobs(production, (vufind_system_type == KRIMDOK ? "krimdok.cronjobs" : "ixtheo_cronjobs"),
```

Comment:
`ixtheo_cronjobs` => should this be `ixtheo.cronjobs`?

Reply:
Yes, indeed!